crates/sui-indexer-alt/src/handlers/sum_coin_balances.rs

```diff
@@ -22,13 +22,6 @@ use crate::{
     schema::sum_coin_balances,
 };
 
-/// Each insert or update will include at most this many rows -- the size is chosen to maximize the
-/// rows without hitting the limit on bind parameters.
-const UPDATE_CHUNK_ROWS: usize = i16::MAX as usize / 5;
```

Comment:
@amnn I saw the `*_CHUNK_ROWS` only exist in these 2 files, while it should be applied to all tables?

Reply:
So having seen how things are set-up, I think we'll want to make use of `FIELD_COUNT` slightly differently -- sorry for the churn:

- We need to handle the integration of `FIELD_COUNT`s differently for concurrent and sequential pipelines. This means we can't do the integration once, in the `Processor` trait. We need to do it in the concurrent and sequential `Handler` traits respectively.
- The `concurrent::Handler` is easiest to integrate with -- it already has a `MAX_CHUNK_ROWS` parameter whose definition we can replace as you've done here, and this parameter is used by the framework automatically.
- You will also need to remove the overrides for `MAX_CHUNK_ROWS` in `concurrent::Handler` impls.
- Sequential pipelines (like this one) don't automatically chunk their rows -- that's done by each implementation of `sequential::Handler::commit`, so for these ones, the best thing to do is to keep the constants per handler implementation. (we also can't say that e.g. all deletions in sequential pipelines need exactly one bind per row).

Reply:
@amnn thanks for the detailed thoughts, I made some changes and now it's ready for another review
- removed the `const`s from `Processor`
- change `MAX_CHUNK_ROWS` of `concurrent::Handler` to use `FieldCount` and remove the overrides in each concurrent handler
- add `MAX_INSERT/DELETE_CHUNK_ROWS` with `FieldCount` and use them in the impl. of `commit` in each `sequential::Handler`.